tensorboard/webapp/plugins/npmi/BUILD

```diff
@@ -56,8 +64,51 @@ py_test(
     srcs_version = "PY2AND3",
     deps = [
         ":summary",
+        ":metadata",
         "//tensorboard:expect_tensorflow_installed",
-        "//tensorboard/backend/event_processing:data_provider",
+        "//tensorboard/compat",
         "//tensorboard/util:test_util",
     ],
 )
+
+py_library(
+    name = "safe_encoder",
+    srcs = ["safe_encoder.py"],
+    srcs_version = "PY2AND3",
+    visibility = [
+        "//visibility:public",
```

Comment:
Let's avoid making this publicly visible; the default private visibility should suffice here.

Reply:
Done in [30a3b091](https://github.com/tensorflow/tensorboard/pull/3799/commits/30a3b0910a88a82eaeac7eb99bd383f7f397e139)